app/src/androidTest/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/splash/SplashScreenTest.kt

```diff
@@ -83,6 +84,7 @@ class SplashScreenTest : BaseAndroidComposeTest() {
     }
 
     @Test
+    @AllowFlaky(attempts = 5)
     fun when_type_password_into_password_text_field_has_text_with_mask() {
```

Comment:
Why is this test flaky in the first place? 🤔

Reply:
Not sure @ryan-conway 😅 , in fact the failing test function changes from time to time. I set it here because the last time it fails, the log points to this method.

Reply:
@kaungkhantsoe All tests pass repeatedly after cleaning, even without this library. Perhaps we can remove it so that we don't tolerate flaky tests.
If a test is flaky, then either the test is set up incorrectly or the code being tested is not really testable, in which case we need to update our code.
CC: @AVI5HEK thoughts?

Reply:
I will try removing in latter PR @ryan-conway 🙏